pkg/glusterfs/controllerserver.go

```diff
@@ -297,6 +278,8 @@ func (cs *ControllerServer) DeleteVolume(ctx context.Context, req *csi.DeleteVol
 		glog.Errorf("Deleting Volume %s failed: %v", req.VolumeId, err)
 		return nil, status.Errorf(codes.Internal, "error deleting volume: %s", err.Error())
 	}
+
+	glog.Infof("Successfully deleted volume %s", volumeID)
```

Comment:
do we need any log level for this one?

Reply:
I'd say no, it's relevant enough at all log levels.